kivakit-core/pom.xml

```diff
@@ -44,6 +44,11 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>kivakit-mixins</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.mastfrog</groupId>
```

Comment:
we need to be careful not to expose users to a second large framework. can we import the code and make it conform to kivakit conventions instead?

Reply:
Why?  I don't want to maintain two copies of this.

Reply:
Anyway, this library pulls in two things - `util-exceptions`, which is just a bunch of `notNull("foo", foo)` stuff and the ever-handy `Exceptions.chuck(someCheckedException)`, and `function` which is primarily a set of lambda interfaces - the throwing variants and multi-argument variants, which are immensely useful.  So it doesn't really involve a lot of code at all.